Server/Python/src/dbs/web/DBSWriterModel.py

```diff
@@ -36,6 +39,16 @@ def __init__(self, config):
         if isinstance(urls, type({})):
             config.database.connectUrl = urls['writer']
 
+        # initialize NATS if requested
+        self.nats = None
+        if hasattr(config, 'use_nats') and config.use_nats:
+            topic = 'cms.dbs'
+            topics = config.nats_topics
+            if not topics:
+                topics = ['%s.topic' % topic]
+            self.nats = NATSManager(config.nats_server, topics=topics, default_topic=topic)
+            print("DBS NATS: %s" % self.nats)
```

Comment:
Please use logger instead of print.

Reply:
Done with this commit: 1141eb47..69ad3504